src/vchan_cmds.c

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (c) 2024 EPAM Systems
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <domain.h>
+#include <vch.h>
+#include <zephyr/kernel.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/shell/shell.h>
+
+LOG_MODULE_REGISTER(vchan_cmd);
+
+#define ECHO_SERVER_THREAD_PRIO    7
+#define PATH_TEMPLATE "/local/domain/%d/echo"
+#define THREAD_NAME "vchan_echo_server"
+
+static struct k_thread echo_thread;
+static struct k_thread *pecho_thread = NULL;
+static struct vch_handle handle;
+
+static K_THREAD_STACK_DEFINE(echo_thread_stack, CONFIG_VCHAN_ECHO_BUFFER_SIZE);
```

Comment:
This should be `VCHAN_THREAD_STACK_SIZE`

Reply:
Fixed